pom.xml

```diff
@@ -101,13 +100,41 @@
             <version>${lucene.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-api</artifactId>
+            <version>2.7</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-core</artifactId>
+            <version>2.7</version>
+            <scope>test</scope>
+        </dependency>
+        <!-- For ES 2.x -->
         <dependency>
             <groupId>org.elasticsearch</groupId>
             <artifactId>elasticsearch</artifactId>
             <version>${es.version}</version>
             <scope>test</scope>
             <type>test-jar</type>
         </dependency>
+        <!-- For ES 5.x (requires Java 8) -->
+        <!--
```

Comment:
nit: I'm assuming the commented out section will be removed before merging.

Reply:
I was going to leave this in as when we move to Java 8 these will come back.

Reply:
Yeah, considering we'll want to change this in a few months, I think it's fine to keep as-is.

Reply:
Actually, these are the only changes required for Java 8.  However, I'll remove these comments for now.